app/js/templates/portfolio.ejs

```diff
@@ -0,0 +1,10 @@
+<div class="col-md-3 col-sm-6 col-xs-12 <%= it.className %>>">
```

Comment:
nice but please rename the file, the name of the file should name the instance of itself... I can't get what is here... All portfolio?